.vscode/settings.json

```diff
@@ -1,24 +1,28 @@
 {
-	"files.exclude": {
-		"**/.deps": true,
-		"**/*_TEST": true,
-		"**/*_TEST.log": true,
-		"**/*_TEST.o": true,
-		"**/*_TEST.trs": true,
-		"**/*.hdf5": true,
-		"**/*.la": true,
-		"**/*.libs": true,
-		"**/*.lo": true,
-		"**/*.log": true,
-		"**/Makefile.in": true,
-		"aclocal.m4": true,
-		"config.h": true,
-		"configure": true,
-		"config.status": true,
-		"libtool": true,
-		"stamp-h1": true,
-		"autom4te.cache": true,
-		"config/": true,
-	},
-	"C_Cpp.intelliSenseCacheSize": 0
+    "files.exclude": {
+        "**/.deps": true,
+        "**/*_TEST": true,
+        "**/*_TEST.log": true,
+        "**/*_TEST.o": true,
+        "**/*_TEST.trs": true,
+        "**/*.hdf5": true,
+        "**/*.la": true,
+        "**/*.libs": true,
+        "**/*.lo": true,
+        "**/*.log": true,
+        "**/Makefile.in": true,
+        "aclocal.m4": true,
+        "config.h": true,
+        "configure": true,
+        "config.status": true,
+        "libtool": true,
+        "stamp-h1": true,
+        "autom4te.cache": true,
+        "config/": true,
+    },
+    "C_Cpp.intelliSenseCacheSize": 0,
+
+    // Insert spaces when pressing Tab. This setting is overriden
```

Comment:
JSON does not allow comments, does it? Does this work? Is VSCode tollerating it?

Reply:
It does :) and it works fine for me